core/thread/id.go

```diff
@@ -369,4 +371,5 @@ type LogInfo struct {
 	PrivKey crypto.PrivKey
 	Addrs   []ma.Multiaddr
 	Head    cid.Cid
+	Direct  bool // Whether log is 'directly' managed (owned). Indirect logs are 'peer' logs.
```

Comment:
Went with Direct here, that way the zero value is false, making it backward compatible.

Reply:
Sweet

Reply:
Opting for Managed as suggested elsewhere. Much better for explanation.